src/usage_metrics/core/github_nonpartitioned.py

```diff
@@ -0,0 +1,77 @@
+"""Transform data from Github logs for stargazer and fork data."""
+
+import os
+
+import pandas as pd
+from dagster import (
+    AssetExecutionContext,
+    asset,
+)
+
+
+@asset(
+    io_manager_key="database_manager",
```

Comment:
If we use the existing `IOManagers` for non partitioned assets will we keep appending the same data to a table?

Reply:
We'll only append any rows with primary keys that aren't already in the data - so in effect, only new rows of the data will be added. The non-partitioned records don't have `partition_key` columns and aren't looking for that when they load data.